config.lua

```diff
@@ -34,31 +22,39 @@ config({'development', 'production'}, {
     mail_server = os.getenv('MAIL_SMTP_SERVER'),
     mail_from_name = "Snap!Cloud",
     mail_from = "postmaster@snap-cloud.cs10.org",
-    mail_footer = "<br/><br/><p><small>Please do not reply to this email. This message was automatically generated by the Snap!Cloud. To contact an actual human being, please write to <a href='mailto:snap-support@bjc.berkeley.edu'>snap-support@bjc.berkeley.edu</a></small></p>",
+    mail_footer = "<br/><br/><p><small>Please do not reply to this email. This message was automatically generated by the Snap!Cloud. To contact an actual human being, please write to <a href='mailto:snap-support@bjc.berkeley.edu'>snap-support@bjc.berkeley.edu</a></small></p>"
+})
 
+config('development', {
+    use_daemon = 'off',
+    site_name = 'dev | Snap Cloud',
+    hostname = 'localhost',
+    port = os.getenv('PORT') or 8080,
+    ssl_cert_name = os.getenv('SSL_CERT_NAME') or 'host',
+    enable_auto_ssl = 'false',
+    num_workers = 1,
+    code_cache = 'off',
+    log_directive = 'stderr notice',
+    dns_resolver = 'localhost',
+    secret = os.getenv('SESSION_SECRET_BASE') or 'this is a secret',
     measure_performance = true
 })
 
 config('production', {
     use_daemon = 'on',
-    postgres = {
-        host = os.getenv('DATABASE_URL'),
-        user = os.getenv('DATABASE_USERNAME'),
-        password = os.getenv('DATABASE_PASSWORD'),
-        database = os.getenv('DATABASE_NAME')
-    },
-    port = port or 80,
+    port = os.getenv('PORT') or 80,
     site_name = 'Snap Cloud',
     hostname = 'snap-cloud.cs10.org',
     ssl_cert_name = os.getenv('SSL_CERT_NAME') or 'snap-cloud.cs10.org',
-    enable_https = true,
     enable_auto_ssl = 'true', -- lapis needs a string
     secret = os.getenv('SESSION_SECRET_BASE'),
     num_workers = 16,
     code_cache = 'on',
 
-    --- TODO: See if we can turn this on without a big hit
-    measure_performance = false,
+    log_directive = 'logs/error.log warning',
```

Comment:
In my local setup this is giving me an error:

`nginx: [emerg] invalid log level "warning" in /home/renard/Code/Lua/snapCloud/nginx.conf.compiled:14`

Reply:
Bleh, you're right. I fixed this for #44, but forgot to migrate the change back to this branch. It should just be `warn`. :)